docs/building.md

```diff
@@ -181,10 +181,10 @@ Make sure to install the workload `Desktop development with C++` and the individ
 
 Install [Git for Windows](https://gitforwindows.org/)
 
-Run the following commands in Command Prompt or Powershell  
-git clone https://github.com/microsoft/vcpkg  
-cd vcpkg  
-bootstrap-vcpkg.bat  
+Run the following commands in Command Prompt or Powershell
+git clone https://github.com/microsoft/vcpkg
+cd vcpkg
+bootstrap-vcpkg.bat
```

Comment:
The spaces here are actually necessary to indicate that consecutive lines are not part of the same paragraph and should, in fact, be placed on separate lines. However, perhaps it would make more sense to put the commands in a code block.

Reply:
Done